### PR TITLE
Tutorial overhaul

### PR DESCRIPTION
**Pull request by Wikunia**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**How did you address these issues with this PR? What methods did you use?**
I explained some things in more detail like what options one have to create an `Action` and how `Scaling` works in the background. In general we should aim to explain more in tutorials instead of just making animations. Currently I find it quite hard to read the tutorials and get the information I need. Hopefully this PR will fix some of those issues.

### CODE REVIEW

**Review by TheCedarPrince — APPROVED** [review]
I added one commit that updated some grammar in tutorials 2 and 5. Also did some light editing to make it flow with my writing style just a bit better. Otherwise, looks great to me!

File: docs/src/tutorials/tutorial_2.md [hunk 1]
```diff
@@ -81,48 +81,62 @@ The following invocation will create the head:
 ...
 ```
 
-### Different types of defining an Action
+An `Action` consists of at least one part, namely calling a function which draw something on to the canvas.
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_2.md:84]: "draw" -> "draws", right? (Always feel awkward to correct your English :smile: )

File: docs/src/tutorials/tutorial_2.md [hunk 2]
```diff
@@ -81,48 +81,62 @@ The following invocation will create the head:
 ...
 ```
 
-### Different types of defining an Action
+An `Action` consists of at least one part, namely calling a function which draw something on to the canvas. 
+An `Action` is fully comprised of `frames`, an optional `id`, a `function` that draws something on a canvas, an optional `Animation`, and an optional `Movement`.
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_2.md:85]: Would start with a different word than "An" and `frames` are kinda optional.

**Review by Wikunia — COMMENTED** [review]
LGTM just some very minor things.

File: docs/src/tutorials/tutorial_2.md (hunk 1 above)
Comment (TheCedarPrince) [reply docs/src/tutorials/tutorial_2.md:84]: Haha, good catch!

**Pull request merged**